lib/livebook_web/live/session_live.ex

```diff
@@ -1796,7 +1796,8 @@ defmodule LivebookWeb.SessionLive do
             id: section.id,
             name: section.name,
             parent: parent_section_view(section.parent_id, data),
-            status: cells_status(section.cells, data)
+            status: cells_status(section.cells, data),
+            definitions: cells_definitions(section.cells, data)
```

Comment:
Looking at this, I realised it would be better to call it `identifier_definitions` everywhere for extra clarity. Could you rename? Sorry!